iaso/api/payments/views.py

```diff
@@ -264,12 +264,17 @@ def _get_table_row(self, payment, row_num=None, export_type="csv"):
         return [
             str(payment.id),
             payment.status,
-            str(payment.user.id),  # Added User ID
+            str(payment.user.id),
             payment.user.username,
+            (
+                payment.user.iaso_profile.phone_number.as_e164
+                if payment.user.iaso_profile and payment.user.iaso_profile.phone_number
+                else None
+            ),
```

Comment:
I think that adding this should've broken something in unit tests.

Reply:
yes me too, tests are not good enough on this